### PR TITLE
[#1] Setup project

### LINKED ISSUE #1: Setup project
## Why

- Create a new empty project for development (Generate new project from [Flutter Template](https://github.com/nimblehq/flutter_templates))

- Update README.md
- Update default repo's url in `.github` templates

## Who Benefits?

Developers

### PR DESCRIPTION
**Pull request by hoangnguyen92dn**
Close #1

## What happened 👀

- Generate new project from `flutter_templates`

## Insight 📝

- Follow instruction from `flutter_templates`, there was an issue with xcode installation on my local

https://apple.stackexchange.com/questions/254380/why-am-i-getting-an-invalid-active-developer-path-when-attempting-to-use-git-a

*Solution:* Execute this command `xcode-select --install`

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+SPEC CHECKSUMS:
+  Flutter: f04841e97a9d0b0a8025694d0796dd46242b2854
+  flutter_config: 2226c1df19c78fe34a05eb7f1363445f18e76fc1
+  integration_test: a1e7d09bd98eca2fc37aefd79d4f41ad37bdbbe5
+  package_info_plus: 6c92f08e1f853dc01228d6f553146438dafcd14e
+  permission_handler: ccb20a9fad0ee9b1314a52b70b76b473c5f8dab0
+
+PODFILE CHECKSUM: de482529810d31c8802b6be0fd93c8c3e038be64
+
+COCOAPODS: 1.11.3
```
Comment (Wadeewee) [inline ios/Podfile.lock:40]: @hoangnguyen92dn, I got the error waring with the `cocoapods: (1.11.3)` when try to build the app on ios here.

Screen Shot 2566-03-07 at 16 27 50

I think we need to install bundle to update gems.

File: ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -0,0 +1,1059 @@
...
+				ENABLE_BITCODE = NO;
+				INFOPLIST_FILE = Runner/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
+				PRODUCT_BUNDLE_IDENTIFIER = co.nimblehq.flutter.ic.survey.staging;
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "match AdHoc co.nimblehq.flutter.template.staging";
```
Comment (Wadeewee) [inline ios/Runner.xcodeproj/project.pbxproj:384]:
```suggestion
				PROVISIONING_PROFILE_SPECIFIER = "match AdHoc co.nimblehq.flutter.ic.survey.staging";
```

File: ios/Runner.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -0,0 +1,1059 @@
...
```
Comment (Wadeewee) [inline ios/Runner.xcodeproj/project.pbxproj:516]:
```suggestion
				PROVISIONING_PROFILE_SPECIFIER = "match AdHoc co.nimblehq.flutter.ic.survey.staging";
```

File: ios/Runner.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -0,0 +1,1059 @@
...
```
Comment (Wadeewee) [inline ios/Runner.xcodeproj/project.pbxproj:544]:
```suggestion
				PROVISIONING_PROFILE_SPECIFIER = "match AdHoc co.nimblehq.flutter.ic.survey.staging";
```

File: ios/Runner.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -0,0 +1,1059 @@
...
```
Comment (Wadeewee) [inline ios/Runner.xcodeproj/project.pbxproj:624]:
```suggestion
				PROVISIONING_PROFILE_SPECIFIER = "match AdHoc co.nimblehq.flutter.ic.survey.staging";
```

File: ios/Runner.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -0,0 +1,1059 @@
...
+				ENABLE_BITCODE = NO;
+				INFOPLIST_FILE = Runner/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
+				PRODUCT_BUNDLE_IDENTIFIER = co.nimblehq.flutter.ic.survey;
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "match AppStore co.nimblehq.flutter.template";
```
Comment (Wadeewee) [inline ios/Runner.xcodeproj/project.pbxproj:706]:
```suggestion
				PROVISIONING_PROFILE_SPECIFIER = "match AppStore co.nimblehq.flutter.ic.survey";
```

File: ios/Runner.xcodeproj/project.pbxproj [hunk 7]
```diff
@@ -0,0 +1,1059 @@
...
+				ENABLE_BITCODE = NO;
+				INFOPLIST_FILE = Runner/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
+				PRODUCT_BUNDLE_IDENTIFIER = co.nimblehq.flutter.ic.survey.staging;
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "match AppStore co.nimblehq.flutter.template.staging";
```
Comment (Wadeewee) [inline ios/Runner.xcodeproj/project.pbxproj:785]:
```suggestion
				PROVISIONING_PROFILE_SPECIFIER = "match AppStore co.nimblehq.flutter.ic.survey.staging";
```

File: ios/Runner.xcodeproj/project.pbxproj [hunk 8]
```diff
@@ -0,0 +1,1059 @@
...
+				ENABLE_BITCODE = NO;
+				INFOPLIST_FILE = Runner/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
+				PRODUCT_BUNDLE_IDENTIFIER = co.nimblehq.flutter.ic.survey;
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "match AppStore co.nimblehq.flutter.template";
```
Comment (Wadeewee) [inline ios/Runner.xcodeproj/project.pbxproj:863]:
```suggestion
				PROVISIONING_PROFILE_SPECIFIER = "match AppStore co.nimblehq.flutter.ic.survey";
```

File: ios/Runner.xcodeproj/project.pbxproj [hunk 9]
```diff
@@ -0,0 +1,1059 @@
...
```
Comment (Wadeewee) [inline ios/Runner.xcodeproj/project.pbxproj:1014]:
```suggestion
				PROVISIONING_PROFILE_SPECIFIER = "match AppStore co.nimblehq.flutter.ic.survey";
```

File: ios/Podfile.lock (hunk 1 above)
Comment (hoangnguyen92dn) [reply ios/Podfile.lock:40]: @Wadeewee Indeed. I pushed the new `Podfile.lock` 🙏

**Review by Wadeewee — APPROVED** [review]

**Review by doannimble — APPROVED** [review]

**Pull request merged**